templates/projects/list.tmpl

```diff
@@ -35,33 +35,36 @@
 				</div>
 			</div>
 		</div>
-		<div class="milestone list">
+		<div class="milestone-list">
 			{{range .Projects}}
-				<li class="item">
-					{{svg .IconName}} <a href="{{.Link}}">{{.Title}}</a>
-					<div class="meta">
-						{{$closedDate:= TimeSinceUnix .ClosedDateUnix $.locale}}
-						{{if .IsClosed}}
-							{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.closed" $closedDate | Safe}}
+				<li class="milestone-card">
+					<h3 class="flex-text-block gt-m-0">
+						{{svg .IconName 16}}
+						<a class="muted" href="{{.Link}}">{{.Title}}</a>
+					</h3>
+					<div class="milestone-toolbar">
+						<div class="group">
+							<div class="flex-text-block">
+								{{svg "octicon-issue-opened" 14}}
+								{{$.locale.PrettyNumber .NumOpenIssues}}&nbsp;{{$.locale.Tr "repo.issues.open_title"}}
+							</div>
+							<div class="flex-text-block">
+								{{svg "octicon-check" 14}}
+								{{$.locale.PrettyNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
+							</div>
+						</div>
+						{{if and $.CanWriteProjects (not $.Repository.IsArchived)}}
+						<div class="group">
+							<a class="flex-text-inline" href="{{.Link}}/edit" data-id={{.ID}} data-title={{.Title}}>{{svg "octicon-pencil" 14}}{{$.locale.Tr "repo.issues.label_edit"}}</a>
```

Comment:
Some nits:

* `data-xxx={{.xxx}}`, the value part should be quoted as `data-xxx="{{.xxx}}"`
* Is `data-id={{.ID}}` necessary? I can see this is a simple link, no idea why it needs `data-id`
    * Maybe there are more similar cases below
* Does it need to use `data-tooltip-content` for tooltip instead of `data-title` ?

----

Oh I see they are from old code, so only "nits".